pallets/lbp/src/lib.rs

```diff
@@ -81,19 +81,25 @@ impl Default for WeightCurveType {
 /// Max weight corresponds to 100%
 pub const MAX_WEIGHT: LBPWeight = 100_000_000;
 
+/// Max sale duration is 14 days, assuming 6 sec blocks
+pub const MAX_SALE_DURATION: u32 = (60 * 60 * 24 / 6) * 14;
```

Comment:
```suggestion
pub const MAX_SALE_DURATION: u32 = 14 * primitives::constants::time::Days;
```

Reply:
those are not relaychain Days, relaychain has 6 sec blocktime